index.html

```diff
@@ -1,63 +1,82 @@
 <!DOCTYPE html>
 <html lang="en">
-<meta charset="utf-8">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="icon" type="image/png" href="data:image/png;base64,">
+<link rel="canonical" href="https://bestmotherfucking.website">
 <title>Best Motherfucking Website</title>
+<meta name="description" content="This website is fucking perfect. Seriously, what the fuck else do you want?">
+<meta property="og:type" content="website" />
+<meta property="og:title" content="Best Motherfucking Website" />
+<meta property="og:description" content="This website is fucking perfect. Seriously, what the fuck else do you want?" />
+<meta property="og:url" content="https://bestmotherfucking.website" />
+<meta property="og:site_name" content="Best Motherfucking Website" />
+<meta property="og:image" content="https://ipfs.io/ipfs/QmeVTyCq1JoN6Vhv5RQxWU28Lg6cQd3E273RwVwQpbX7Be?filename=og.png" />
```

Comment:
ipfs: no thanks

Reply:
also wtf is that image even, stop

Reply:
The image adds nothing, it's an external dependency, it's bytes.

It actually makes the preview worse, take up more space!

![image](https://user-images.githubusercontent.com/38327267/154839775-607c6421-67cc-4ddf-9b5e-a7bcd8340011.png)

![image](https://user-images.githubusercontent.com/38327267/154839933-562735b5-0442-4f84-9813-b4270252df59.png)

Reply:
edit:

> Let's live in the future

no thanks, here, old boring is better